derive_builder/src/options/mod.rs

```diff
@@ -2,14 +2,15 @@
 //!
 //! Attribute parsing occurs in multiple stages:
 //!
-//! 1. Builder options on the struct are parsed into `OptionsBuilder<StructMode>`.
-//! 1. The `OptionsBuilder<StructMode>` instance is converted into a starting point for the
-//!    per-field options (`OptionsBuilder<FieldMode>`) and the finished struct-level config,
-//!    called `StructOptions`.
-//! 1. Each struct field is parsed, with discovered attributes overriding or augmenting the
-//!    options specified at the struct level. This creates one `OptionsBuilder<FieldMode>` per
-//!    struct field on the input/target type. Once complete, these get converted into
-//!    `FieldOptions` instances.
+//! 1. Builder options on the struct are parsed into
```

Comment:
@azriel91 please undo the formatting on comments; this has broken the ordered list rendering.